tests/unit/views/PasswordCheck.spec.js

```diff
@@ -38,18 +38,28 @@ describe('PasswordCheck', () => {
         await setPasswordTo('a');
       });
 
-      it.todo('should highlight one letter is satisfied');
-      it.todo('should not highlight number satisfied');
+      it('should highlight one letter is satisfied', () => {
+        expect(wrapper.get('[data-test-rule="OneLetter"]').classes('is-valid')).toBe(true);
```

Comment:
It's better not to have more than one definition of the same entity:
```suggestion
        expect(wrapper.get(`[data-test-rule="${RULE.OneLetter}"]`).classes('is-valid')).toBe(true);
```